src/components/AnchorForCommentsOnly/BaseAnchorForCommentsOnly/index.js

```diff
@@ -13,27 +13,19 @@ import fileDownload from '../../../libs/fileDownload';
 /*
  * This is a default anchor component for regular links.
  */
-const BaseAnchorForCommentsOnly = ({
-    href,
-    rel,
-    target,
-    children,
-    style,
-    fileName,
-    ...props
```

Comment:
I'm not sure if this matters for this case, but might be worth noting that before this was setting `props` to the "rest" of the arguments and now `props` will be everything. Might want to make sure we are not overwriting any values in places where these props are spread and the use of "rest" syntax is intentional.

Reply:
This is fixed now.